idb/idb.go

```diff
@@ -28,6 +31,9 @@ type TxnRow struct {
 	// TxnBytes is the raw signed transaction with apply data object.
 	TxnBytes []byte
 
+	// RootTxnBytes the root transaction raw signed transaction with apply data object, only inner transactions have this.
+	RootTxnBytes []byte
```

Comment:
Coalesce could be used to return RootTxnBytes / TxnBytes in the same parameter, but that could make it easy to mix up the other TxnRow